telegram_bot/bot.py

```diff
@@ -13,3 +17,28 @@ def send_message(chat_id, text):
     )
     response.raise_for_status()
     return response.json()
+
+
+def send_message_to_prague_channel():
+    events = get_future_events("https://pyvo.cz/api/series/praha-pyvo.ics")
+    for event in events:
+        summary = event["summary"].replace("(", "").replace(")", "")
```

Comment:
What does this do?

Reply:
The summary is in the format `Next week! ('19.10.2022', 'Pražské Pyvo – nepotvrzeno; tradiční termín srazu')`, so this removes`(` & `)`. I see that it's confusing, but I've solved the problem few times in this way, so now I'm not quite, if this is the standard way or only my quick&dirty solution? 😆

Reply:
I've asked the question, because:

- It's strange this is needed, I didn't understand why it's needed.
- If it's needed, for most of the reasons, this solution would be prone to errors

Imagine if the title of the meetup looked like this:

```
Pražské Pyvo – Všechno (nejen) o instalování balíčků (nejen) pro začátečníky
```

Your code would modify the title in a way you probably don't want.

Reply:
That's a great observation! I've addressed it in the PR https://github.com/pyvec/pyvo-bot/pull/2.